Explorer/Assets/DCL/LOD/Systems/UpdateSceneLODInfoSystem.cs

```diff
@@ -145,7 +145,7 @@ private void CheckSceneReadinessAndClean(ref SceneLODInfo sceneLODInfo, SceneDef
         {
             if (sceneLODInfo.CurrentLOD.LodKey.Level == 0)
```

Comment:
Hey @dalkia what this condition means? Could we add a variable so its more descriptive on reading?

Reply:
We are basically checking if the lod level is 0. If so, it means that the player may have landed in that scene, therefore we need to mark it as ready in the scene cache.
I will create a method named IsLOD0 for clarity